.dev_scripts/gather_models.py

```diff
@@ -5,9 +5,7 @@
 python gather_models.py ${root_path} ${out_dir}
 
 Example:
-python gather_models.py \
-work_dirs/pgd_r101_caffe_fpn_gn-head_3x4_4x_kitti-mono3d \
-work_dirs/pgd_r101_caffe_fpn_gn-head_3x4_4x_kitti-mono3d
+python gather_models.py ./work_dirs ./gather
```

Comment:
So the updated version can automatically collect all the models under work_dirs? May need to add more comments as a hint of how to use this script.

Reply:
I provide a `mmdet3d model release doc` in our feishu documentation for details. Of course, I'll add more comments here as hint of how to use this script.